partition_snapshot_reader.hh

```diff
@@ -234,24 +235,27 @@ class partition_snapshot_flat_reader : public flat_mutation_reader::impl, public
             });
         }
 
-        mutation_fragment_opt next_range_tombstone(const query::clustering_range& ck_range,
+        mutation_fragment_opt next_range_tombstone(const query::clustering_range& ck_range_snapshot,
```

Comment:
Can you rename the `ck_range` param in `next_row` to either `ck_range_snapshot` or `ck_range_query` (whichever is needed there) so it's easier to understand which range we want?

Also, it would be nice to add a comment somewhere in this class - maybe before the definition of `next_row` - what `ck_range_snapshot` and `ck_range_query` means and how they are different.